src/Microsoft.AspNetCore.Antiforgery/AntiforgeryOptions.cs

```diff
@@ -19,7 +19,8 @@ public class AntiforgeryOptions
         private CookieBuilder _cookieBuilder = new CookieBuilder
         {
             SameSite = SameSiteMode.Strict,
-            HttpOnly = true
+            HttpOnly = true,
+            SecurePolicy = CookieSecurePolicy.None
```

Comment:
It would be good to have a comment here explaining the issues. We don't anyone auditing this code to see `Secure = None` and have alarms go off.

Reply:
How does the following look?
```
Chrome recently deprecated the support of non-secure endpoints to set cookies with a 'secure' flag or overwrite cookies whose 'secure' flag is set (http://httpwg.org/http-extensions/draft-ietf-httpbis-cookie-alone.html). Since mixing secure and non-secure endpoints is a common scenario in applications, we are relaxing the restriction on secure policy on some cookies by setting to None. However, cookies related to authentication or authorization would still have restrictions as usual.
```
cc @Eilon

Reply:
A few small edits:
```
Some browsers do not allow non-secure endpoints to set cookies with a 'secure' flag or overwrite cookies whose 'secure' flag is set (http://httpwg.org/http-extensions/draft-ietf-httpbis-cookie-alone.html). Since mixing secure and non-secure endpoints is a common scenario in applications, we are relaxing the restriction on secure policy on some cookies by setting to 'None'. Cookies related to authentication or authorization use a stronger policy than 'None'.
```